### ISSUE TITLE
A parallel new cult. The cult of the sea?

### ISSUE THREAD

**Issue by angeltxilon** [Azanor/thaumcraft-suggestions#69]
Thaumcraft will have shortly a fifth and sixth anniversary, yet their bases are very similar to the first edition of the mod.

Currently everything is based on the cult crimson. You make your first magic tricks, then you study the crimson technology, and your adventure finally ends up killing the little crimson Eldritch gods.

I think it's time to add a new parallel cult (ie, antagonized but at the same time eldritch) to crimson cult. Perhaps a "cult of the sea" would fit perfectly in relation to the benthic beasts and pelagic gods normally related to Eldritch things (and sea guardians?).
This could make the game more interesting, making to the thaumaturgic technology of all (or almost all) the fields ramify into 2 or 3 ends.

Add branched goals is something that usually add difficulty and makes the game more interesting, so 2 ways of magic and thaumaturgy (borned of the basic thaumaturgy) separated but equally interdependent way would make the game last longer.

**Comment by Argis13** [comment]
Oh. So is this a Jedi versus Sith kind of thing? With the sea cult wanting to use the vis for good and the crimson for power, with you destined to end the battle however you choose?

**Comment by angeltxilon** [comment]
No, there would not be "goods and bads". Simply they would be two different cults that "does not have the same god/boss and ideology" and therefore tends to avoid or attack themself mutually (due to territorial, religious or power reasons). It is like in the real life, nor goods nor bads, only different societies (in this case cults) with different objectives, ideology and leaders.

Basically what matters are "the gods" or bosses. 
Crimson cult has eldritch humanoid small gods while the cult of the sea could have a more "cthulhunian" god (this mean, a more alien god, of acuatic type). Obviously each cult will have different considerations and activities due to this.

**Comment by Argis13** [comment]
But if you kill the gods, the put their differences aside and the countdown begins. P.S. Sorry for misunderstanding what you meant.

**Comment by slg407** [comment]
basically a dagon cult (nailed it lovecraft)

**Comment by angeltxilon** [comment]
LoL, I just discovered that the cult that I have proposed is both the name of a movie... ïa ïa Cthulhu fhtagn.

Well, the idea stay, basically I suggest to add a new parallel cult for add more diversity to the mod, either a sect of the sea or something else related to eldricht.

**Comment by carso150** [comment]
I like the idea, one cult (the crimson) serve to outher eldrich abominations while they try to learn and understand more about them while another cult (maybe we can name it aquamarine?) serve to gods that came from the seas like marine people, you know giant humanoid crabs, octopuses, fishes, etc, I like the idea hope azanor see it.
The only thing is that is going to be really difficult to add a new cult to the mod, god the crimson cult is not finished

**Comment by l-Luna** [comment]
And with all the sea-based stuff in 1.13, this would fit.
But 'sea' isn't quite as evil as 'crimson'.

**Comment by fingersome** [comment]
THE HALCYON HARBINGERS
... or something

**Comment by fingersome** [comment]
Instead of eldrich shamblers with crabs on their heads they could have nasty squid faced fiends

**Comment by fingersome** [comment]
I’m imagining barnacle encrusted tombs with a portal to a cave world likethe nether but where everything is covered in saltwater abs seaweed and where slimes and scaled things hold dominion
